content/docs/gto/get-started-dvc.md

```diff
@@ -0,0 +1,115 @@
+# Get Started DVC
+
+To leverage concepts of Model and Data Registries in a more explicit way, you
+can denote the `type` of each output. This will let you browse models and data
+separately, address them by `name` in `dvc get`, and eventually, see them in DVC
+Studio.
+
+Let's start with marking an artifact as data or model.
+
+If you're using `dvc add` to track your artifact, you'll need to run:
+
+```dvc
+$ dvc add mymodel.pkl --type model
+```
+
+If you're producing your models in DVC pipeline, you'll need to add
+`type: model` to `dvc.yaml` instead:
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - data.xml
+    outs:
+      - mymodel.pkl:
+          type: model # like this
```

Comment:
to make it symmetrical with plots, params, etc- why don't we make it:

```yaml
models:
   mymodel.pkl
```
do we plan / need other artifact types? (data can be covered by `outs` I think)

Reply:
QQ: in this case, do you assume other fields can be specified like:
```yaml
models:
   mymodel.pkl:
     name: particles-classifier
     description: best hits
     labels:
        - boson
        - higgs
```
?

For now it looks like `type: model` may be enough. I can introduce this `models: ` section OFC (and I'd like us to do this, since `artifacts.yaml` is almost like it, which will make a transition easier for existing users with something like `models: artifacts.yaml` - we also have a user asking if we could allow this to have a separate file), but then we need to take into account how we deal with duplication/overwrites if we allow `models` to be specified in `models: ` section and in outs (`type: model`) simultaneously.

Reply:
A top-level `dvc.yaml` section means we need to do some more work on the DVC side instead of using what we have.

If we go this direction, the biggest question is whether to call it `models:` or make it generic like `registry:` or `artifacts:`. The appeal of `models` is that we start small, but I worry it doesn't make much sense long-term because:
1. There's no model-specific functionality in DVC now or in current plans.
2. People already use DVC as a data registry, and it's inevitable people will want to log non-model artifacts.
3. It provides flexibility for custom types (not sure if this is good or bad).

Reply:
Thanks for ideas. I've updated the doc - please TAL.

Reply:
I think the `dvc.yaml` schema can look like this:

```
# dvc.yaml
stages:
  train:
    cmd: python train.py
    deps:
      - data.xml
    outs:
      - models/mymodel.pkl
artifacts:
  - def-detector:
      path: models/mymodel.pkl
      type: model
      desc: glass defect image classifier
      labels:
        - algo=cnn
        - owner=aguschin
        - project=prod-qual-002
  - models/othermodel.pkl # If no path provided, use name as the path
```
 
This would reuse most of what we already have between:
1. Existing metadata fields for type, desc, labels.
2. Top-level fields we have for plots, metrics, and params.
3. GTO artifacts.yaml (artifacts are in a dict format in artifacts.yaml but a list format above; we could support both to make it less error-prone, which we already do for top-level plots).

I think we can mostly leave alone everything already implemented and only add this new top-level section. I would suggest that only things listed in `artifacts:` (and in artifacts.yaml if we keep it) show up in the Studio registry. Until we get to a non-model registry, we could hide artifacts without `type: model`.

Reply:
It's in the right direction,I like it. by I would make it even simpler initially and do the first class citizen top level `models` section. I don't see a reason to overgeneralize everything at the moment tbh.

Reply:
I think there are good reasons to generalize types. Besides those mentioned above in this thread and by users, we already support `type` in DVC metadata and GTO `artifacts.yaml`, so it fits with what we have already. Is there a good enough reason to rebuild to make it model-specific?